superset/views/dashboard/api.py

```diff
@@ -229,9 +211,9 @@ class DashboardRestApi(DashboardMixin, BaseSupersetModelRestApi):
     }
     filter_rel_fields_field = {"owners": "first_name", "slices": "slice_name"}
 
-    @expose("/<model_id>", methods=["PUT"])
+    @expose("/<pk>", methods=["PUT"])
```

Comment:
is there some reason `model_id` was changed back to `pk`? I think `model_id` here was nice, but if there is a convention to use `pk` then disregard.

Reply:
Yes, under the wood all other ModelRestApi's were exposing an OpenAPi spec with `pk`, so for the sake of consistency I changed it back